frontend/src/Student/components/ShowCard.js

```diff
@@ -18,86 +19,160 @@ const Card = styled.div`
 `
 
 const PhotoThumbnail = styled.img`
-    height: auto;
-    width: auto;
-    min-width: 4em;
-    max-height: 10em;
-    max-width: 100%;
+  height: auto;
+  width: auto;
+  min-width: 4em;
+  max-height: 10em;
+  max-width: 100%;
 `
 
 const EntryNoThumbContainer = styled.div`
   height: 100%;
   padding: 15px;
 `
 
-const EntryThumb = ({entry}) => {
+const EntryContainer = styled.div``
+const JudgingPhase = styled.div`
+  position: relative;
+  border: 1px solid transparent;
+  border-radius: 0.25rem;
+  color: #856404;
+  background-color: #fff3cd;
+  border-color: #ffeeba;
+`
+const Accepted = styled.div`
```

Comment:
I would make all of these labels the same width – in the image you attached, make them all the size of `JudgingPhase`, or make them all be 100% width of their container (or in the case of photos, same width as the photo – which I think is 100% of the container)

Reply:
Width is now the width of the col

![image](https://user-images.githubusercontent.com/6333152/36871564-821f5ea6-1d57-11e8-8776-47d9029dae3c.png)